tools/system_libs.py

```diff
@@ -1215,6 +1215,7 @@ def get_base_name(self):
 
   def get_cflags(self):
     cflags = super(libstandalonewasm, self).get_cflags()
+    cflags += ['-DNDEBUG']
```

Comment:
Hmm.. seems like maybe we need a way to do this consistently across all the system library.  Doesn't have to be part of this change though.